.gitignore

```diff
@@ -4,3 +4,4 @@ Cargo.lock
 *.swp
 *DS_Store
 *.patch
+*NVChip
```

Comment:
I got this too! I think it is created somehow when running the Docker image

Reply:
No, that's the persistent part of the `tpm_server`, so if you have problems related to DA locking, you have to remove it. Might add a line in `all.sh` to remove this file when finished.